myhdl/_Signal.py

```diff
@@ -26,6 +26,8 @@
 negedge -- callable to model a falling edge on a signal in a yield statement
 
 """
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
I've taken care of division in jck@d78bb2ccbe828a1f3cda9acc53f29832a6204b5f but have not included it in this PR since it is not exactly 'low hanging fruit'.